operator/pkg/resources/configuration/patch.go

```diff
@@ -145,6 +145,14 @@ func PropertiesEqual(
 	l logr.Logger, v1, v2 interface{}, metadata rpadmin.ConfigPropertyMetadata,
 ) bool {
 	log := l.WithName("PropertiesEqual")
+
+	// "null" (string) is the only way to define null as the desired value.
+	// Therefore, treat `"null"` as equal to `null` (only if `null`
```

Comment:
Could you rephrase? The end of the sense is miss leading

Reply:
apparently i did not save the file. updated the comment